tests/dragonfly/server_family_test.py

```diff
@@ -101,3 +103,196 @@ def gen_test_data():
         assert cur == 0
         assert len(keys) == 1
         assert keys[0] == key
+
+
+@pytest.mark.asyncio
```

Comment:
I believe all this are unit tests that should be in server_family_test.cc
We use python tests when we must start a client and server to check the functionality, but this is not the case here

Reply:
converted to unit tests. only omitted two cases: blocking command + tests involving real connections.